src/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -438,15 +480,14 @@ private static int FromWin32Value(int value)
         }
 #endif
 
-#if FEATURE_SYSTEM_EVENTS
-        private static void OnUserPreferenceChanging(object sender, UserPreferenceChangingEventArgs e)
+        private static void OnUserPreferenceChanging(object sender, EventArgs args)
         {
-            if (e.Category == UserPreferenceCategory.Color && s_colorTable != null)
+            // Invoking the Delegate.
```

Comment:
nit: comment is not particularly useful.  Consider something along the lines of "Access args.Category value through cached delegate".  Also you can assert that the delegate is not null as well.  If you were able to hook the event you should have been able to initialize the delegate.